{"name": "辅助阅卷测试", "status": "passed", "description": "测试辅助阅卷流程", "steps": [{"name": "登录系统", "status": "passed", "attachments": [{"name": "login_success", "source": "ad85c4ed-ef0c-442d-bb81-d600c72dbedc-attachment.png", "type": "image/png"}], "start": 1743063295660, "stop": 1743063309398}, {"name": "点击辅助阅卷", "status": "passed", "attachments": [{"name": "auxiliary_reading_clicked", "source": "31a0d7cc-2ae4-47f1-9121-89e6053fbe40-attachment.png", "type": "image/png"}], "start": 1743063309398, "stop": 1743063311553}, {"name": "点击庭审笔录1", "status": "passed", "start": 1743063311553, "stop": 1743063313631}, {"name": "点击设为庭审笔录", "status": "passed", "start": 1743063313631, "stop": 1743063314742}, {"name": "输入原告处理意见1", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"E:\\AutoTest\\FgAI\\test_001\\AssistedG.py\", line 251, in test_assisted_reading\n    opinion1 = self.wait.until(\n  File \"D:\\Python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 137, in until\n    value = method(self._driver)\n  File \"D:\\Python39\\lib\\site-packages\\selenium\\webdriver\\support\\expected_conditions.py\", line 110, in _predicate\n    return driver.find_element(*locator)\n  File \"D:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 898, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"D:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 427, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"D:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 404, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"D:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 428, in _request\n    response = self._conn.request(method, url, body=body, headers=headers, timeout=self._client_config.timeout)\n  File \"D:\\Python39\\lib\\site-packages\\urllib3\\request.py\", line 78, in request\n    return self.request_encode_body(\n  File \"D:\\Python39\\lib\\site-packages\\urllib3\\request.py\", line 170, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n  File \"D:\\Python39\\lib\\site-packages\\urllib3\\poolmanager.py\", line 376, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"D:\\Python39\\lib\\site-packages\\urllib3\\connectionpool.py\", line 703, in urlopen\n    httplib_response = self._make_request(\n  File \"D:\\Python39\\lib\\site-packages\\urllib3\\connectionpool.py\", line 449, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"D:\\Python39\\lib\\site-packages\\urllib3\\connectionpool.py\", line 444, in _make_request\n    httplib_response = conn.getresponse()\n  File \"D:\\Python39\\lib\\http\\client.py\", line 1347, in getresponse\n    response.begin()\n  File \"D:\\Python39\\lib\\http\\client.py\", line 307, in begin\n    version, status, reason = self._read_status()\n  File \"D:\\Python39\\lib\\http\\client.py\", line 268, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"D:\\Python39\\lib\\socket.py\", line 704, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1743063314742, "stop": 1743063323901}], "start": 1743063295660, "stop": 1743063323961, "uuid": "c61d9031-4902-4b49-b84d-150c971028d8", "testCaseId": "bcbc7cf4b4f79be87767f13c16480802", "fullName": "test_001.AssistedG.TestAssistedReading#test_assisted_reading"}